### ISSUE TITLE
Syncing, but not Scrobbling

### ISSUE THREAD

**Issue by Kiplol** [trakt/Plex-Trakt-Scrobbler#284]
I'm able to manually sync, and able to see my newly watched movies reflected on the Trakt dashboard.  I'm not able to scrobble, however.
I've gone through the scrobbling FAQ, and I believe I've set things up correctly (The scrobbling checkbox is enabled).

This is my log file, beginning immediately before I begin watching a movie.

```
2016-02-29 11:27:41,046 - plugin.sync.main                 (109301000) :  INFO (plugin.sync.main:150) - (2) Done
2016-02-29 11:27:45,148 - com.plexapp.plugins.trakttv      (109901000) :  INFO (com.plexapp.plugins.trakttv:131) - Saved the dictionary file
2016-02-29 11:30:34,247 - plex_activity.sources.s_logging.parsers.base (108d8d000) :  DEBUG (plex_activity.sources.s_logging.parsers.base:59) - break on "Feb 29, 2016 11:30:33 [0x10a316000] DEBUG - Downloading document http://127.0.0.1:32400/library/metadata/10348"
2016-02-29 11:30:44,568 - plex_activity.sources.s_logging.parsers.base (108d8d000) :  DEBUG (plex_activity.sources.s_logging.parsers.base:59) - break on "Feb 29, 2016 11:30:43 [0x109104000] DEBUG - Completed: [192.168.1.158:62138] GET /:/timeline?ratingKey=10348&key=%2Flibrary%2Fmetadata%2F10348&state=playing&playQueueItemID=637&time=69620&duration=6708960 (5 live) TLS GZIP 16ms 231 bytes 200"
2016-02-29 11:30:54,183 - plex_activity.sources.s_logging.parsers.base (108d8d000) :  DEBUG (plex_activity.sources.s_logging.parsers.base:59) - break on "Feb 29, 2016 11:30:53 [0x109104000] DEBUG - Completed: [192.168.1.158:62155] GET /:/timeline?ratingKey=10348&key=%2Flibrary%2Fmetadata%2F10348&state=playing&playQueueItemID=637&time=79860&duration=6708960 (5 live) TLS GZIP 50ms 231 bytes 200"
2016-02-29 11:30:57,890 - plex_activity.sources.s_logging.parsers.base (108d8d000) :  DEBUG (plex_activity.sources.s_logging.parsers.base:59) - break on "Feb 29, 2016 11:30:56 [0x10a210000] DEBUG - Found session GUID of tl7g6kqoy9g7rpb9 in session start."
2016-02-29 11:31:37,727 - plex_activity.sources.s_logging.parsers.base (108d8d000) :  DEBUG (plex_activity.sources.s_logging.parsers.base:59) - break on "Feb 29, 2016 11:31:36 [0x10cb2b000] DEBUG - Downloading document http://127.0.0.1:32400/library/metadata/10348"
```

Here's another log.  It should be of me opening the channel's settings, closing them, then playing a movie.

```
2016-02-29 11:34:26,721 - com.plexapp.plugins.trakttv      (109901000) :  DEBUG (com.plexapp.plugins.trakttv:717) - Handling request GET /video/trakt/:/prefs
2016-02-29 11:34:26,725 - com.plexapp.plugins.trakttv      (109901000) :  DEBUG (com.plexapp.plugins.trakttv:814) - Found route matching /video/trakt/:/prefs
2016-02-29 11:34:26,739 - com.plexapp.plugins.trakttv      (109901000) :  DEBUG (com.plexapp.plugins.trakttv:88) - Sending packed state data (108 bytes)
2016-02-29 11:34:26,739 - com.plexapp.plugins.trakttv      (109901000) :  DEBUG (com.plexapp.plugins.trakttv:924) - Response: [200] MediaContainer, 3735 bytes
2016-02-29 11:34:26,740 - root                             (109901000) :  INFO (root:1611) - 304 GET /video/trakt/:/prefs (127.0.0.1) 19.00ms
2016-02-29 11:40:07,668 - com.plexapp.plugins.trakttv      (109901000) :  DEBUG (com.plexapp.plugins.trakttv:717) - Handling request GET /video/trakt/:/prefs
2016-02-29 11:40:07,671 - com.plexapp.plugins.trakttv      (109901000) :  DEBUG (com.plexapp.plugins.trakttv:814) - Found route matching /video/trakt/:/prefs
2016-02-29 11:40:07,685 - com.plexapp.plugins.trakttv      (109901000) :  DEBUG (com.plexapp.plugins.trakttv:88) - Sending packed state data (108 bytes)
2016-02-29 11:40:07,685 - com.plexapp.plugins.trakttv      (109901000) :  DEBUG (com.plexapp.plugins.trakttv:924) - Response: [200] MediaContainer, 3735 bytes
2016-02-29 11:40:07,686 - root                             (109901000) :  INFO (root:1611) - 304 GET /video/trakt/:/prefs (127.0.0.1) 18.85ms
2016-02-29 11:40:16,602 - com.plexapp.plugins.trakttv      (109901000) :  DEBUG (com.plexapp.plugins.trakttv:717) - Handling request GET /video/trakt/:/prefs/set?level_pyemitter=1
2016-02-29 11:40:16,605 - com.plexapp.plugins.trakttv      (109901000) :  DEBUG (com.plexapp.plugins.trakttv:814) - Found route matching /video/trakt/:/prefs/set
2016-02-29 11:40:16,607 - com.plexapp.plugins.trakttv      (109901000) :  DEBUG (com.plexapp.plugins.trakttv:198) - Saved the user preferences
2016-02-29 11:40:16,611 - requests.packages.urllib3.connectionpool (109901000) :  INFO (requests.packages.urllib3.connectionpool:207) - Starting new HTTP connection (1): localhost
2016-02-29 11:40:16,616 - requests.packages.urllib3.connectionpool (109901000) :  DEBUG (requests.packages.urllib3.connectionpool:387) - "GET / HTTP/1.1" 200 744
2016-02-29 11:40:16,617 - plugin.modules.migrations.account (109901000) :  INFO (plugin.modules.migrations.account:251) - Plex Token: anonymous
2016-02-29 11:40:16,647 - plugin.models.m_plex.account     (109901000) :  DEBUG (plugin.models.m_plex.account:149) - Refreshing anonymous plex account
2016-02-29 11:40:16,647 - plugin.models.m_plex.credential  (109901000) :  DEBUG (plugin.models.m_plex.credential:89) - Refreshing anonymous plex credential
2016-02-29 11:40:16,652 - plugin.modules.migrations.account (109901000) :  DEBUG (plugin.modules.migrations.account:205) - Ignoring basic credential update for <TraktAccount username: u'Kiplol'>, already exists (ConstraintError: UNIQUE constraint failed: trakt.credential.basic.account_id)
2016-02-29 11:40:16,659 - plugin.preferences.main          (109901000) :  DEBUG (plugin.preferences.main:49) - Updated 'activity_mode' option in database
2016-02-29 11:40:16,660 - plugin.preferences.options.matcher (109901000) :  DEBUG (plugin.preferences.options.matcher:32) - Configured matcher, extended: True
2016-02-29 11:40:16,661 - plugin.preferences.main          (109901000) :  DEBUG (plugin.preferences.main:49) - Updated 'matcher' option in database
2016-02-29 11:40:16,680 - plugin.preferences.main          (109901000) :  DEBUG (plugin.preferences.main:49) - Updated 'sync_watched' option in database
2016-02-29 11:40:16,681 - plugin.preferences.main          (109901000) :  DEBUG (plugin.preferences.main:49) - Updated 'sync_watchlist' option in database
2016-02-29 11:40:16,683 - plugin.preferences.main          (109901000) :  DEBUG (plugin.preferences.main:49) - Updated 'sync_ratings' option in database
2016-02-29 11:40:16,686 - plugin.managers.m_trakt.account  (109901000) :  DEBUG (plugin.managers.m_trakt.account:112) - PIN hasn't changed, ignoring account authorization update
2016-02-29 11:40:16,686 - plugin.preferences.main          (109901000) :  DEBUG (plugin.preferences.main:49) - Updated 'pin' option in database
2016-02-29 11:40:16,688 - plugin.preferences.main          (109901000) :  DEBUG (plugin.preferences.main:49) - Updated 'sync_clean_collection' option in database
2016-02-29 11:40:16,689 - plugin.preferences.main          (109901000) :  DEBUG (plugin.preferences.main:49) - Updated 'sync_playback' option in database
2016-02-29 11:40:16,691 - plugin.preferences.main          (109901000) :  DEBUG (plugin.preferences.main:49) - Updated 'sync_run_library' option in database
2016-02-29 11:40:16,694 - plugin.preferences.main          (109901000) :  DEBUG (plugin.preferences.main:49) - Updated 'sync_run_interval' option in database
2016-02-29 11:40:16,695 - plugin.preferences.main          (109901000) :  DEBUG (plugin.preferences.main:49) - Updated 'sync_collection' option in database
2016-02-29 11:40:16,697 - plugin.preferences.main          (109901000) :  DEBUG (plugin.preferences.main:49) - Updated 'sync_liked_lists' option in database
2016-02-29 11:40:16,698 - plugin.preferences.main          (109901000) :  DEBUG (plugin.preferences.main:49) - Updated 'sync_ratings_conflict' option in database
2016-02-29 11:40:16,700 - plugin.preferences.main          (109901000) :  DEBUG (plugin.preferences.main:49) - Updated 'start_scrobble' option in database
2016-02-29 11:40:16,702 - plugin.preferences.main          (109901000) :  DEBUG (plugin.preferences.main:49) - Updated 'sync_personal_lists' option in database
2016-02-29 11:40:16,704 - plugin.core.helpers              (109901000) :  DEBUG (plugin.core.helpers:19) - Spawned thread with name 'None'
2016-02-29 11:40:16,706 - com.plexapp.plugins.trakttv      (109901000) :  DEBUG (com.plexapp.plugins.trakttv:88) - Sending packed state data (108 bytes)
2016-02-29 11:40:16,706 - com.plexapp.plugins.trakttv      (109901000) :  DEBUG (com.plexapp.plugins.trakttv:924) - Response: [200] MessageContainer, 104 bytes
2016-02-29 11:40:16,707 - plugin.core.logger.main          (109d04000) :  DEBUG (plugin.core.logger.main:123) - Changed 'pyemitter' logger level to DEBUG
2016-02-29 11:40:16,708 - root                             (109901000) :  INFO (root:1611) - 200 GET /video/trakt/:/prefs/set?level_pyemitter=1 (127.0.0.1) 106.26ms
2016-02-29 11:40:28,976 - plex_activity.sources.s_logging.parsers.base (108d8d000) :  DEBUG (plex_activity.sources.s_logging.parsers.base:59) - break on "Feb 29, 2016 11:40:28 [0x10a210000] DEBUG - Downloading document http://127.0.0.1:32400/library/metadata/12849"
2016-02-29 11:40:28,976 - pyemitter                        (108d8d000) :  DEBUG (pyemitter:46) - [plex_activity.sources.s_logging.parsers.now_playing.NowPlayingParser]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '12849', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '0', 'duration': '7293376', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '62575'},), kwargs: {})
2016-02-29 11:40:28,977 - pyemitter                        (108d8d000) :  DEBUG (pyemitter:46) - [plex_activity.sources.s_logging.main.Logging]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '12849', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '0', 'duration': '7293376', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '62575'},), kwargs: {})
2016-02-29 11:40:28,977 - pyemitter                        (108d8d000) :  DEBUG (pyemitter:46) - [plex_activity.activity.Activity   ]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '12849', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '0', 'duration': '7293376', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '62575'},), kwargs: {})
```

**Comment by Kiplol** [comment]
Also possibly worth mentioning: After waiting some time, the Plex user on http://trakt-for-plex.github.io/configuration/#/configuration/accounts?id=1 changes from my Plex username (ekipper) to Administrator.

**Comment by Kiplol** [comment]
In the FAQ, it says I should look in the log for something like this:

```
2016-01-23 15:38:16,649 - DEBUG - Filters.is_valid_client({'platform': 'Konvergo', 'product': 'Plex Media Player', 'key': '...', 'title': '...'})
2016-01-23 15:38:16,650 - DEBUG - [scrobble_clients] no preference found
2016-01-23 15:38:16,650 - DEBUG - [filter_networks] IPv4Address(u'...') - wildcard
2016-01-23 15:38:16,665 - DEBUG - Activity matched against rule: None
2016-01-23 15:38:16,708 - DEBUG - [scrobble_names] validate - value: 'fuzeman', allow: ['fuzeman'], deny: []
2016-01-23 15:38:16,709 - DEBUG - Activity matched against rule: <UserRule priority: 1, account: 1, name: u'fuzeman'>
```

But I can't seem to find anything in mine resembling that (not even the word "Scrobble," outside of the name of the plugin).

```
2016-03-01 14:31:01,606 - plex_activity.sources.s_logging.parsers.base (10921c000) :  DEBUG (plex_activity.sources.s_logging.parsers.base:59) - break on "Mar 01, 2016 14:31:00 [0x10a9a8000] DEBUG - It took 0.0 sec to serialize a list with 0 elements."
2016-03-01 14:31:01,640 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.sources.s_logging.parsers.now_playing.NowPlayingParser]:emit(event: 'logging.playing', args: ({'user_id': None, 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': None, 'time': '0', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': None, 'port': '57824'},), kwargs: {})
2016-03-01 14:31:01,641 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.sources.s_logging.main.Logging]:emit(event: 'logging.playing', args: ({'user_id': None, 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': None, 'time': '0', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': None, 'port': '57824'},), kwargs: {})
2016-03-01 14:31:01,642 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.activity.Activity   ]:emit(event: 'logging.playing', args: ({'user_id': None, 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': None, 'time': '0', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': None, 'port': '57824'},), kwargs: {})
2016-03-01 14:31:10,818 - plex_activity.sources.s_logging.parsers.base (10921c000) :  DEBUG (plex_activity.sources.s_logging.parsers.base:59) - break on "Mar 01, 2016 14:31:10 [0x109181000] DEBUG - Completed: [192.168.1.158:57835] GET /:/timeline?ratingKey=10348&key=%2Flibrary%2Fmetadata%2F10348&state=playing&playQueueItemID=642&time=9916&duration=6708960 (9 live) TLS GZIP 6ms 231 bytes 200"
2016-03-01 14:31:10,819 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.sources.s_logging.parsers.now_playing.NowPlayingParser]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '9916', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '57835'},), kwargs: {})
2016-03-01 14:31:10,820 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.sources.s_logging.main.Logging]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '9916', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '57835'},), kwargs: {})
2016-03-01 14:31:10,820 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.activity.Activity   ]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '9916', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '57835'},), kwargs: {})
2016-03-01 14:31:21,392 - plex_activity.sources.s_logging.parsers.base (10921c000) :  DEBUG (plex_activity.sources.s_logging.parsers.base:59) - break on "Mar 01, 2016 14:31:20 [0x109204000] DEBUG - Completed: [192.168.1.158:57836] GET /:/timeline?ratingKey=10348&key=%2Flibrary%2Fmetadata%2F10348&state=playing&playQueueItemID=642&time=19675&duration=6708960 (9 live) TLS GZIP 4ms 231 bytes 200"
2016-03-01 14:31:21,392 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.sources.s_logging.parsers.now_playing.NowPlayingParser]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '19675', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '57836'},), kwargs: {})
2016-03-01 14:31:21,393 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.sources.s_logging.main.Logging]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '19675', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '57836'},), kwargs: {})
2016-03-01 14:31:21,393 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.activity.Activity   ]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '19675', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '57836'},), kwargs: {})
2016-03-01 14:31:31,379 - plex_activity.sources.s_logging.parsers.base (10921c000) :  DEBUG (plex_activity.sources.s_logging.parsers.base:59) - break on "Mar 01, 2016 14:31:30 [0x109204000] DEBUG - Completed: [192.168.1.158:57838] GET /:/timeline?ratingKey=10348&key=%2Flibrary%2Fmetadata%2F10348&state=playing&playQueueItemID=642&time=29936&duration=6708960 (9 live) TLS GZIP 4ms 231 bytes 200"
2016-03-01 14:31:31,380 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.sources.s_logging.parsers.now_playing.NowPlayingParser]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '29936', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '57838'},), kwargs: {})
2016-03-01 14:31:31,381 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.sources.s_logging.main.Logging]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '29936', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '57838'},), kwargs: {})
2016-03-01 14:31:31,381 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.activity.Activity   ]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '29936', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '57838'},), kwargs: {})
2016-03-01 14:31:40,946 - plex_activity.sources.s_logging.parsers.base (10921c000) :  DEBUG (plex_activity.sources.s_logging.parsers.base:59) - break on "Mar 01, 2016 14:31:40 [0x109204000] DEBUG - Completed: [192.168.1.158:57840] GET /:/timeline?ratingKey=10348&key=%2Flibrary%2Fmetadata%2F10348&state=playing&playQueueItemID=642&time=39919&duration=6708960 (9 live) TLS GZIP 4ms 231 bytes 200"
2016-03-01 14:31:40,956 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.sources.s_logging.parsers.now_playing.NowPlayingParser]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '39919', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '57840'},), kwargs: {})
2016-03-01 14:31:40,957 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.sources.s_logging.main.Logging]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '39919', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '57840'},), kwargs: {})
2016-03-01 14:31:40,957 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.activity.Activity   ]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '39919', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '57840'},), kwargs: {})
ntifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '57845'},), kwargs: {})
2016-03-01 14:32:32,221 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.sources.s_logging.main.Logging]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '90679', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '57845'},), kwargs: {})
2016-03-01 14:32:32,221 - pyemitter                        (10921c000) :  DEBUG (pyemitter:46) - [plex_activity.activity.Activity   ]:emit(event: 'logging.playing', args: ({'user_id': '1', 'address': '192.168.1.158', 'ratingKey': '10348', 'state': 'playing', 'client': 'Plex Web (Chrome)', 'time': '90679', 'duration': '6708960', 'machineIdentifier': '279614d5-a4ab-482c-8fc3-f412c025ba34', 'user_name': 'ekipper', 'port': '57845'},), kwargs: {})
 
```

**Comment by fuzeman** [comment]
I'm wondering if there is an error happening on startup which is causing the scrobbler to ignore all your watching activity.

-------------------

Could you do the following:
 - Restart PMS
 - Wait a minute for the plugin to startup completely
 - Post the latest plugin log file (via pastebin, or dropbox, etc..)

**Comment by Kiplol** [comment]
Thanks for the response!  I will do this tonight when I get home.

**Comment by Kiplol** [comment]
Got it.  I restarted PMS twice because who knows.

http://pastebin.com/BX9ye18X

**Comment by fuzeman** [comment]
```
2016-03-03 18:32:41,112 : DEBUG - Method 'websocket' has been disabled
```

Have you changed the "Activity mode" option to "Logging" for legacy client support (Plex Media Center)? if not, changing this option back to "Automatic" would be a good idea.

--------------

I'm guessing there must be a bug with the "Logging" activity method, will need to do some testing to work out why it isn't working..

**Comment by Kiplol** [comment]
Ah, you are certainly right.  I had tried fiddling with options and forgot to switch that back.  Sorry about that.  Here's a new log with activity mode set to automatic.

http://pastebin.com/Eb2cW3yb

**Comment by fuzeman** [comment]
Could you test the changes in this version of the plugin: https://github.com/fuzeman/Plex-Trakt-Scrobbler/archive/issue/284.zip

Post another log file if the scrobbler still isn't working.

**Comment by Kiplol** [comment]
Sorry to say it still wasn't scrobbling :(

http://pastebin.com/cm2m4X4s

**Comment by fuzeman** [comment]
Great :smile:, will include this fix in the next release.

**Comment by fuzeman** [comment]
Fix released in [v1.0.1.2](https://github.com/trakt/Plex-Trakt-Scrobbler/releases/tag/v1.0.1.2)
